utils.py

```diff
@@ -405,13 +412,23 @@ def get_time_in_millisecs(datetime_obj):
 
 
 def get_current_time_in_millisecs():
-    """Returns time in milliseconds since the Epoch."""
+    """Returns time in milliseconds since the Epoch.
+
+    Returns:
+       float. Returns time in millisecond.
+    """
     return get_time_in_millisecs(datetime.datetime.utcnow())
 
 
 def get_human_readable_time_string(time_msec):
     """Given a time in milliseconds since the epoch, get a human-readable
     time string for the admin dashboard.
+
+    Args:
+        time_msec: float. Passing time in milliseconds.
+
+    Returns:
+        str. Time in human readable format.
```

Comment:
A string representing time in a human readable format.

Reply:
done

Reply:
I don't see any change here*